ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -430,6 +430,10 @@ host_monitoring_cron:
   minute: "1"
   job: ops-runner -f -s 30 -n ccexp.openshift.master.certificates /usr/bin/cron-certificate-expirations --cert-list=/etc/origin/master,/etc/origin/node
 
+- name: "Report running builds" 
+  minute: "*/5"
+  job: /usr/bin/cron-send-build-counts
```

Comment:
Ideally, you should run this under ops-runner and have the script `set -e` somewhere so that if any of the sends break, we'll know about it. Otherwise, if something breaks, we probably won't notice until something else goes wrong.